jormungandr-lib/src/interfaces/reward_parameters.rs

```diff
@@ -11,12 +11,18 @@ pub enum RewardParams {
         ratio: Ratio,
         epoch_start: Epoch,
         epoch_rate: NonZeroU32,
+        rewards_limit: Ratio,
+        npools: NonZeroU32,
+        npools_threshold: NonZeroU32,
```

Comment:
These value should be optionals.

Reply:
rewards_limit can be set to "100/100" to have no effect, so there is no need to make this optional.
npools and npools threshold can be both set to zero to have no effect.  They should be present in the actual testnet, but could conceivably be made optionals.

Reply:
optional is way better for the genesis file.

Reply:
OK.